scss/_mixins.scss

```diff
@@ -7,26 +7,26 @@
 	width: 100%;
 
 	.o-expander__toggle {
-		@include oTypographySans($scale: 2, $progressive: true);
+		@include oTypographySans($scale: 2);
 		margin-top: 12px;
 		i {
-			@include oIconsGetIcon('arrow-down', getColor('teal-40'), 12);
-			@include oIconsBaseStyles();
+			@include oIconsContent('arrow-down', oColorsByName('teal-40'), 12);
+			@include oIconsContentBaseStyles();
 			display: inline-block;
 			padding: 0 12px;
 			box-sizing: border-box;
 			&:hover {
-				@include oIconsGetIcon('arrow-down', getColor('black-80'), 12);
+				@include oIconsContent('arrow-down', oColorsByName('black-80'), 12);
 			}
 		}
 		&:hover {
-			color: getColor('black-80');
+			color: oColorsByName('black-80');
 			text-decoration: none;
 		}
 		&[aria-expanded="true"] i {
-			@include oIconsGetIcon('arrow-up', getColor('teal-40'), 12);
+			@include oIconsContent('arrow-up', oColorsByName('teal-40'), 12);
```

Comment:
I really want to swap this with `rotate(180deg)`

Reply:
<img width="759" alt="Screenshot 2019-11-26 at 15 11 03" src="https://user-images.githubusercontent.com/10405691/69646013-016bc900-105f-11ea-83e8-668d7f3d6301.png">